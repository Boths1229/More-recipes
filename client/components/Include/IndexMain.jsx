import React from 'react';

const IndexMain = () => {
  return (
    <div className="container">
      <main className="row">
        <div className="col-sm-12 col-md-12 headline text-center">
          <h1 className="heading">
            Share your amazing recipe with the world
          </h1>
          <h6 className="brief">
            More-recipe is built to allow users,
            share their amazing recipe
           and learn new recipes from different part of the world.
          </h6>
          <div className="quick-access">
            <a href="./html/signnup.html">
              <button className="btn btn-create">
                Create an account
              </button>
            </a>
            <p className="text-inline">
              Already on More-recipes?
              <a href="./html/signin.html">Sign in</a>
            </p>
          </div>
        </div>
      </main>
    </div>
  );
};

export default IndexMain;
