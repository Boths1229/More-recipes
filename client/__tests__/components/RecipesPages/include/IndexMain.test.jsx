import React from 'react';
import { shallow } from 'enzyme';
import IndexMain from '../../../../components/Include/IndexMain';

describe('Index main component', () => {
  it('matches snapshot', () => {
    const output = shallow(<IndexMain />);
    expect(output).toMatchSnapshot();
  });
});
