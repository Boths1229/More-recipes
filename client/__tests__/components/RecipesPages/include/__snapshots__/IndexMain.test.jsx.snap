// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index main component matches snapshot 1`] = `
<div
  className="container"
>
  <main
    className="row"
  >
    <div
      className="col-sm-12 col-md-12 headline text-center"
    >
      <h1
        className="heading"
      >
        Share your amazing recipe with the world
      </h1>
      <h6
        className="brief"
      >
        More-recipe is built to allow users, share their amazing recipe and learn new recipes from different part of the world.
      </h6>
      <div
        className="quick-access"
      >
        <Link
          replace={false}
          to="/signup"
        >
          <button
            className="btn btn-create"
          >
            Create an account
          </button>
        </Link>
        <p
          className="text-inline"
        >
          Already on More-recipes?
          <Link
            replace={false}
            to="/signin"
          >
              Sign in
          </Link>
        </p>
      </div>
    </div>
  </main>
</div>
`;
